docs/HighLevelDesign.md

```diff
@@ -81,3 +81,38 @@ their subgraph.
 * [Add a new graph
 transform](../include/onnxruntime/core/optimizer/graph_transformer.h)
 * [Add a new rewrite rule](../include/onnxruntime/core/optimizer/rewrite_rule.h)
+
+## The ONNX Runtime and Windows OS integration
+
+The ONNX runtime shipped with the Windows operating system in build 1809 (RS5).  The runtime was embedded inside the Windows.AI.MachineLearning.dll and was exposed via that WinRT API (WinML for short).  It includes CPU support and a DirectML execution provider for GPU support.   Since then it has continued to ship in every version of Windows.
+
+Starting with the 1.2 release nuget we are bringing a new layered architecture to the ONNX Runtime and Windows ML.
```

Comment:
We should be able to remove the word "nuget" here. This is true for both source and zip.